lib/rindle.js

```diff
@@ -146,17 +146,14 @@ exports.extract = function(stream, callback) {
  *   console.log('All files written!');
  * });
  */
-exports.bifurcate = function(stream, output1, output2, callback) {
-  'use strict';
-
-  stream.pipe(output1);
-  stream.pipe(output2);
-
-  // Fulfil once we finish writing to both output streams
-  return Promise.all([
-    exports.wait(output1),
-    exports.wait(output2)
-  ]).nodeify(callback);
+exports.bifurcate = function (stream, output1, output2, callback) {
+	stream.pipe(output1);
+	stream.pipe(output2);
+
+	// Fulfil once we finish writing to both output streams
```

Comment:
```suggestion
	// Fulfill once we finish writing to both output streams
```

Reply:
Technically both are used depending on flavor of English, I think you're right that fulfill is the more common (only?) one in American English though

Reply:
TIL!

From my computer's dictionary:

> fulfil | fʊlˈfɪl | (US fulfill)